src/type/scalars.js

```diff
@@ -117,24 +117,36 @@ export const GraphQLFloat = new GraphQLScalarType({
   },
 });
 
-function serializeString(value: mixed): string {
-  // Support serializing objects with custom valueOf() functions - a common way
-  // to represent an complex value which can be represented as a string
-  // (ex: MongoDB id objects).
-  const result =
-    value && typeof value.valueOf === 'function' ? value.valueOf() : value;
+// Support serializing objects with custom toJSON() or valueOf() functions -
+// a common way to represent an complex value which can be represented as
```

Comment:
grammar nit: "a complex value" instead of "an complex value"

Reply:
Thanks 👍 
Fixed